core-bundle/src/Resources/config/services.yml

```diff
@@ -466,6 +466,13 @@ services:
         calls:
             - ["setName", ["contao_frontend"]]
 
+    contao.slug.helper:
+        class: Contao\CoreBundle\Slug\Helper
```

Comment:
I think the naming rules want the class to be called `SlugHelper` if you want to go for helper (similar to `StringUtil`). It could just be `Contao\CoreBundle\Slug` though?

Reply:
Done in c6f8790d4b3be2d3fd07210725662fc807a46425